flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/publisher/polling/PollingRecordPublisher.java

```diff
@@ -46,6 +48,8 @@
 
 	private static final Logger LOG = LoggerFactory.getLogger(PollingRecordPublisher.class);
 
+	private static final SequenceNumber TIMESTAMP_SENTINEL_SEQUENCE_NUMBER = SENTINEL_AT_TIMESTAMP_SEQUENCE_NUM.get();
```

Comment:
I don't think it's necessary to define a static final field here, as enums already static and final by default.
i.e. it'll be equivalent to just use the enum `SENTINEL_AT_TIMESTAMP_SEQUENCE_NUM` in the new changes.